### ISSUE TITLE
Do the ARIA TABS 1.2 fail WCAG 1.4.10 Reflow?

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/aria#1718]
When using the zoom instructions from the Understanding Success Criterion [1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) setting a screen width of 1280px @400% browser zoom the Tabpanel content is truncated and horizontal scrolling occurs.

My personal testing demonstrates that the Tabs do reflow or stack, however the Tabpanel content does not reflow.

ALT=ARIA Tabs example zoomed and content is cut off.
![Tabs-Only-zoomed](https://user-images.githubusercontent.com/4565139/162103478-ff908a9e-c986-4aec-8ca9-4c5edd822895.png)

ALT=Showing Chrome Developer tools where I have removed all other page content so that only the Tab widget is available.
![Tabs-Only-Code](https://user-images.githubusercontent.com/4565139/162103463-c8744f02-737d-443b-9b78-7117ce2bc7c3.png)

**Comment by JAWS-test** [comment]
I think that's a border case. The element itself is exactly 320 px wide (tablist+tabpanel), but with the padding from the body it happens that not the tabpanel has to be scrolled, but the whole page to get the tabpanel completely in the visible area.

**Comment by chlane** [comment]
At VMware, we tackled this with an overflow Tabs design as seen in the "Overflow Tabs"  example http://www.chrislane.info/clarity3demo1/. The tradeoff in this implementation, is that there is a button that only sighted mouse users need.
